docker/README.md

```diff
@@ -0,0 +1,66 @@
+# armadietto [![Build Status](https://secure.travis-ci.org/remotestorage/armadietto.svg)](http://travis-ci.org/remotestorage/armadietto) [![js-semistandard-style](https://img.shields.io/badge/code%20style-semistandard-brightgreen.svg?style=flat-square)](https://github.com/Flet/semistandard) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0eaafdf96ebb47a9ac462bcf6a7ccb06)](https://www.codacy.com/app/lesion/armadietto?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=remotestorage/armadietto/&amp;utm_campaign=Badge_Grade)
+
+> ### :warning: WARNING
+> Please do not consider `armadietto` production ready, this project is still
+> considered experimental.  As with any alpha-stage storage technology, you
+> MUST expect that it will eat your data and take precautions against this. You
+> SHOULD expect that its APIs and storage schemas will change before it is
+> labelled stable.
+
+## What is this?
+
+Armadietto is a [RemoteStorage](https://remotestorage.io) server written for Node.js.
+
+This is a complete rewrite of [reStore](https://github.com/jcoglan/restore).
+
+It is also available as the
+[armadietto](https://www.npmjs.com/package/armadietto) NPM package.
+
+## Usage
+
+```
+docker run -d -p 8000:8000 remotestorage/armadietto:latest
```

Comment:
Under MacOS 11.6.2, with Docker Desktop 4.4.2, this line alone evokes the error
`Unable to find image 'remotestorage/armadietto:latest' locally`
Should there be a build step?

Reply:
Yes. Will work once it has been published, but yes, will add info about building.

Reply:
Added in [a53db93](https://github.com/remotestorage/armadietto/pull/37/commits/a53db93cb5151f7d440670d493bf7b8ed6e519a1)

Reply:
Have just added info about running the docker image behind a proxy as well as it is quite a common thing to do [723f843](https://github.com/remotestorage/armadietto/pull/37/commits/723f843bcfdaba66f4210745ef93fd60b73d3b82)